backend/controllers/iepHistory.js

```diff
@@ -10,6 +10,7 @@ const getIepHistoryFactory = elite2Api => {
     // calls to the database.
     const uniqueUserIds = [...new Set(iepSummary.iepDetails.map(details => details.userId))]
     const uniqueAgencyIds = [...new Set(iepSummary.iepDetails.map(details => details.agencyId))]
+    const levels = [...new Set(iepSummary.iepDetails.map(details => details.iepLevel))]
```

Comment:
Could possibly extract out a separate variable that explicitly ignores the first detail?:
`const [summaryDetail, ...iepDetails] = iepSummary.iepDetails` 

It might be a bit more explicit, and then you could refer to `iepDetails` throughout and avoid the mutation of state with the `shift`.

Reply:
Yeap, have done this, thank you!